README.md

```diff
@@ -59,12 +61,17 @@ func main() {
 
 	c.SubscribeTopics([]string{"myTopic", "^aRegex.*[Tt]opic"}, nil)
 
-	for {
-		msg, err := c.ReadMessage(-1)
+	// A signal handler or similar could be used to set this to false to break the loop.
+	run := true
+
+	for run {
+		msg, err := c.ReadMessage(time.Second)
 		if err == nil {
 			fmt.Printf("Message on %s: %s\n", msg.TopicPartition, string(msg.Value))
-		} else {
+		} else if err.(kafka.Error).Code() != kafka.ErrTimedOut {
```

Comment:
Perfect :+1:

We should make a convenience function on the Error type, so you can do:  `if err.(Kafka.Error).IsTimeout() {`